### DISCUSSION TITLE
ESP32S3 - Low default BTU stack size

### DISCUSSION THREAD

**Discussion by BillyGriffiths** [espressif/arduino-esp32#9679] [Q&A] [answered]
I am busy porting a project from ESP32 to ESP32S3 and I'm running into bluetooth stack size issues. 

`ESP32` sdkconfig has `CONFIG_BT_BTU_TASK_STACK_SIZE=8192`
`ESP32S3` sdkconfig has `CONFIG_BT_BTU_TASK_STACK_SIZE=4096`

@me-no-dev Looking at https://github.com/espressif/esp32-arduino-lib-builder/commit/44d1eb6cae61c64bfc83931abc4fc671e657d9d2 it seems the common value has been updated to 8192 for all models. Will this change eventually filter through to `/arduino-esp32`? If not, how should I go about changing the stack size?

Thanks!!

**Comment by me-no-dev** [comment, marked as answer]
3.0.0 will bring that feature. You can try 3.0.0-RC3 or wait till Monday for final to be released

**Reply by BillyGriffiths** [reply]
That is **excellent** news. Thank you @me-no-dev

**Comment by BillyGriffiths** [comment]
Released! Thanks everyone (and especially you @me-no-dev) for your hard work in making this happen!
